app/reducers/icoState.ts

```diff
@@ -4,19 +4,13 @@ import { IcoPhase, LOAD_ICO_PARAMS, NEW_PHASE_ACTION } from "../actions/constant
 
 export interface IcoState {
   loading: boolean;
-  address: string;
   startDate: string;
   endDate: string;
   minCap: number;
   maxCap: number;
   icoPhase: IcoPhase;
 }
 
-interface IStandardReduxAction<T> {
```

Comment:
Did you decide to take off the `IStandardReduxAction`?

Reply:
It's moved to types.ts file